src/phoenix/db/models.py

```diff
@@ -117,7 +117,6 @@ class Project(Base):
     __table_args__ = (
         UniqueConstraint(
             "name",
-            sqlite_on_conflict="IGNORE",
```

Comment:
Just confirming my understanding, is "IGNORE" the default behavior for unique constraints?

Reply:
(for Postgres and SQLite)

Reply:
No. Removing this line just so the logic is not duplicated in two places.

Reply:
default behavior is to raise exception
```
dbapi2.IntegrityError: UNIQUE constraint failed
```